rskj-core/src/main/java/co/rsk/jsonrpc/JsonRpcError.java

```diff
@@ -28,6 +29,8 @@ public class JsonRpcError implements JsonRpcResultOrError {
 	public static final int METHOD_NOT_FOUND = -32601;
 	public static final int INVALID_PARAMS = -32602;
 	public static final int INTERNAL_ERROR = -32603;
+	public static final int RESPONSE_LIMIT_ERROR = -32011;
```

Comment:
```suggestion
	public static final int RPC_LIMIT_ERROR = -32011;
```

... as this is used for both JsonRpcResponseLimitError & JsonRpcTimeoutError?